jedi/plugins/stdlib.py

```diff
@@ -790,10 +861,37 @@ def wrapper(cls, metaclasses):
                 return [DictFilter({
                     name.string_name: EnumInstance(cls, name).name for name in filter_.values()
                 })]
+
+            if metaclass.py__name__() == 'ModelBase' \
+                    and metaclass.get_root_context().py__name__() == 'django.db.models.base':
+                django_dict_filter = new_django_dict_filter(cls)
+                if django_dict_filter is not None:
+                    return django_dict_filter
```

Comment:
Please put all of the code you wrote into `jedi/plugins/django.py`. This get_metaclass_filters function can also be copied (obviously without the EnumMeta stuff).